lldb/include/lldb/Symbol/ObjectFile.h

```diff
@@ -689,6 +690,8 @@ class ObjectFile : public std::enable_shared_from_this<ObjectFile>,
   virtual llvm::StringRef
   GetReflectionSectionIdentifier(swift::ReflectionSectionKind section);
 
+  virtual bool CanContainSwiftReflectionData(const Section &section);
```

Comment:
I see we assert false in the implementation why don't we make it pure virtual using `=0` instead? I see we use a mix in this class. Detecting we have not implemented an `override` at runtime seems less ideal.

Reply:
I gave it a default behavior of `return false`, instead of making it pure virtual.